_docs/15c-permissions-example.md

```diff
@@ -210,3 +254,44 @@ export const SERVICE_CONFIG: Object = {
 };
 ```
 
+Other important configuration it's to define on the `main-routing.module.ts` the `canActivateChild` parameter to manage the routes permissions.
```

Comment:
I think it's a good idea that the example includes restrictPermissionsRedirect: '403' and a screen with the redirect